src/pks/energy/energy_base_pk.cc

```diff
@@ -288,6 +290,8 @@ void EnergyBase::SetupEnergy_(const Teuchos::Ptr<State>& S)
       ->AddComponent("cell", AmanziMesh::Entity_kind::CELL, 1);
   }
 
+  decoupled_from_subsurface_ = plist_->get<bool>("decoupled from subsurface", false); //surface-only system
```

Comment:
Don't worry too much about this, but in an ideal world, the MPC that couples surface star would set this flag in the child's parameter list to avoid making the user do it.  We do that for things like "coupled_to_surface_via_flux_" where the MPCCoupledWater sets that.  If it is possible, you may consider doing that to avoid making the user do it.